components/Banner/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { AppLayout } from '@/components';
```

Comment:
You can delete the file itself. It's no. needed.

Reply:
done